.vscode/tasks.json

```diff
@@ -0,0 +1,50 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Run tox",
```

Comment:
Might be nice later to expose just `tox lint` or something, but this is a simple example for now.